src/Command/InstallCommand.php

```diff
@@ -198,13 +193,21 @@ private function setupDatabase()
     {
         $this->io->section('Step 2 of 4: Setting up database.');
 
+        $conn = $this->entityManager->getConnection();
+        $databasePlatform = $conn->isConnected() ? $conn->getDatabasePlatform() : null;
+
         // user want to reset everything? Don't care about what is already here
         if (true === $this->defaultInput->getOption('reset')) {
             $this->io->text('Dropping database, creating database and schema, clearing the cache');
 
+            $this->runCommand('doctrine:schema:drop', ['--force' => true, '--full-database' => true]);
+
+            if (!$databasePlatform instanceof PostgreSQLPlatform) {
```

Comment:
Why?

Reply:
because PostgreSQL always create the database, and you can't delete the database you're connected too